cmd/loop/loopout.go

```diff
@@ -41,7 +41,7 @@ var loopOutCommand = cli.Command{
 		},
 		cli.Uint64Flag{
 			Name:  "amt",
-			Usage: "the amount in satoshis to loop out",
+			Usage: "the amount in satoshis to loop out, consult the loop terms command to get the minimum & maximum loop amounts supported by the server.",
```

Comment:
We generally try to keep our lines below 80 characters see https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md

I personally have a 'ruler' set up in my editor, which shows me exactly where the 80 character limit is set up (you can google that for your editor of choice).

See the above flag `addr` as example how to break the lines.